### PR TITLE
Hotfix/mark job successful on spark app end

### PR DESCRIPTION
**Pull request by qqmbr4k**

### CODE REVIEW

**Comment by pferrel** [comment]
@andreyshikov has this been merged with develop?

**Comment by andreyshikov** [comment]
> @andreyshikov has this been merged with develop?

@pat no, it's a PR to develop. I asked you in slack to take a look at README.md and config.yml changes because they are not in develop for some reason... Can you please take a look?

File: .circleci/config.yml [hunk 1]
```diff
@@ -114,6 +114,13 @@ jobs:
             case "${CIRCLE_BRANCH}" in
               "master")
                 docker image push actionml/harness:latest
+
+                docker tag actionml/harness:latest actionml/harness:master-$SHORT_GIT_HASH
+                docker image push actionml/harness:master-$SHORT_GIT_HASH
+                
+                HARNES_VERSION=$(cat version)
+                docker tag actionml/harness:latest actionml/harness:$HARNES_VERSION
+                docker image push actionml/harness:$HARNES_VERSION
```
Comment (pferrel) [inline .circleci/config.yml:123]: @andreyshikov why do we need to change this? Isn't the config.yml in develop and master correct already? 

If this needs to be reviewed, maybe you should ask Dioni

Comment (andreyshikov) [reply .circleci/config.yml:123]: this PR is not up to date, deleting

Comment (andreyshikov) [reply .circleci/config.yml:123]: I'll remove all unrelated, thanks

**Pull request closed without merging**